### PR TITLE
issue #511 resolved - draft text to welcome newcomers to dashboard added using modal

### PR DESCRIPTION
**Pull request by Gauravano**
Hi, I have tried to solve issue #511 by adding a modal as the content is long so I tried this. The modal will open if the user is just a week old for the website.It can be closed by crossing the window or by clicking out of modal. Here;s the preview of modal added.
welcome_text

If you want me to handle this issue using normal notification like this, kindly respond.Thank You.
check1

### CODE REVIEW

**Comment by jywarren** [comment]
Hi, I see what you're saying; do you think we could make the WELCOME header just bold, but on the same paragraph as the rest? That should compact it. 

Also, I think `asking a question` and `exploring projects` could just be links to their respective URLs, rather than on bullet points. What do you think? I guess I prefer to compact this instead of to use a modal. 

Thanks!!

**Comment by jywarren** [comment]
Hi, just checking if you'd had a chance to look at my suggestions? This is very close, thank you!

**Comment by Gauravano** [comment]
hi, @jywarren, your suggestions are very useful and I have tried to work on them. Kindly check the screenshot and tell for more optimisations.Thank You.
update

**Comment by jywarren** [comment]
That looks really good. Can you put the two links at the bottom in buttons, so for example:

```html
<p>
  <a class="btn btn-sm btn-default" href="...">Asking a question</a> or 
  <a class="btn btn-sm btn-default" href="...">Exploring projects</a>
</p>
```

And could you push a new commit to this branch with your new code? Super and thanks for making the additional changes!

File: app/views/dashboard/dashboard.html.erb [hunk 1]
```diff
@@ -1,3 +1,32 @@
+<% if Time.now - current_user.created_at < 1.week %>
```
Comment (jywarren) [inline app/views/dashboard/dashboard.html.erb]: Also, you'll probably need to add an `if current_user && ` before this to only attempt it if you're logged in. That's what's causing the failure in the tests!

**Comment by Gauravano** [comment]
check2pull
Buttons like this or any change required? Also, thanks for clearing about the existence of current_user

**Comment by jywarren** [comment]
That looks perfect! Thanks!

File: app/assets/javascripts/dashboard.js [hunk 2]
```diff
@@ -1,3 +1,25 @@
+$(document).ready(function() {
```
Comment (jywarren) [inline app/assets/javascripts/dashboard.js]: Hi, is the modal code here still needed? It seems like the alert box approach may not require the modal code. Thanks!!

Comment (Gauravano) [reply app/assets/javascripts/dashboard.js]: Yes, this code is redundant now .Thanks for pointing and I will take care of these types of mistakes for future contributions.

**Comment by Gauravano** [comment]
Please help me to figure out why checks are failing as I have only removed redundant code in my last commit. Thank you.

**Comment by jywarren** [comment]
Ok, this was confusing to me too, but i see in the logs:

> `raise_if_we_cannot_find_the_commit': Commit 3e4f22c7 doesn't exist. Are you running `danger local/pr` against the correct repository? Also this usually happens when you rebase/reset and force-pushed. (RuntimeError)

On this line - https://travis-ci.org/publiclab/plots2/builds/286597109?utm_source=github_status&utm_medium=notification

I'm going to try restarting the build. You can do this yourself by closing and reopening the PR. Sometimes this happens if two pushes are made in quick succession... Not sure why exactly here but it doesn't seem to be related to your code. 

Sorry this was a bit frustrating!

**Comment by jywarren** [comment]
Looks like it worked! I'd like to ask @steviepubliclab and @ebarry to take a final look at the text on Monday before merging in. But your work looks great! Thanks so much!

**Comment by jywarren** [comment]
And just to be clear @ebarry @steviepubliclab  -- this message is shown to everyone who's signed up less than 1 week ago. If there are any text tweaks you'd like to do now, please ping in! 

This is just a starting point. We can always add/edit/expand later, in addition to using this as an example for future first-time coders of how we can add more triggers for different time periods.

**Comment by ebarry** [comment]
+1 great!

**Comment by jywarren** [comment]
Super -- merging now, thank you!!!

**Comment by steviepubliclab** [comment]
Looks good to me too!

**Comment by Gauravano** [comment]
Thank you @jywarren @ebarry @steviepubliclab

**Pull request merged**